src/Footer.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+function Footer(props) {
+  return (
+    <div className="footer">
+      <ul className="footer-info">
+        {props.contact.map(info => {
+          return (
+            <li className="info" key={info}>
```

Comment:
the key should always be unique, so it's recommended to use the index, e.g. `key={index}`. Can you remember how we can access the index when using `map`?